src/lightbox/Lightbox.js

```diff
@@ -49,6 +49,7 @@ export default function Lightbox(props: Props) {
 
   // Pulled out here just because this function is used twice.
   const handleImagePress = useCallback(() => {
+    LayoutAnimation.configureNext(LayoutAnimation.create(100, 'easeInEaseOut', 'opacity'));
```

Comment:
It'd be good to be clear that this doesn't just reimplement the old animation -- it replaces it with a different animation.

In particular the old animation had the header and footer slide in from the top and bottom, while this one has them just fade in.

It's a shame that (AFAICT) this LayoutAnimation API doesn't seem to support translation at all -- I think it would not be possible to implement an animation like the old one using this API.

I kind of like the slide-in/out animation. But it looks like both Google's and Apple's "Photos" apps have the top and bottom bars fade in and out, with no sliding; so even though I'm not sure I like that quite as much, it's clearly a perfectly fine way to do it.

Reply:
... Hmm, and now I look back at your (animated) screenshots, and it looks like it still is doing slide-in/out! I like the effect -- but I sure am confused about what that `'opacity'` argument is doing. According to the doc:
https://reactnative.dev/docs/layoutanimation
it should be "the layout property to animate"; but in your screencaps it looks like `top` and `bottom`, or really the overall position that they together work out to, is animating, and `opacity` isn't at all.

Reply:
Huh, yeah, odd—I actually don't see anything happening with the opacity. I took `Presets.easeInEaseOut` and just adjusted the duration from 300 to 100, since it seemed kind of long. (The new revision will be clearer about this.)

Reply:
Not sure what `Presets.easeInEaseOut` was doing with 'opacity'.